### DISCUSSION TITLE
Menu

### DISCUSSION THREAD

**Discussion by AyoubSayah** [chakra-ui/chakra-ui#5762] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/57968451/159464654-6956136a-44db-4452-ba87-3a1a79875d69.png)
i want to pass to menu wiidth 100% , to take the width of parent container,it will be good if i can pass some props to popover

**Comment by chr-ge** [comment]
Have you tried the `matchWidth` prop?

> If true, the popper will match the width of the reference at all times. It's useful for autocomplete, `date-picker` and select patterns.

**Comment by AyoubSayah** [comment]
yap i tried
![image](https://user-images.githubusercontent.com/57968451/159704861-dacb2e86-320f-4313-a3f6-5c5fdf4a091d.png)
![image](https://user-images.githubusercontent.com/57968451/159705155-7a59a5d8-aabb-44a5-b9d0-cef3ffd10414.png)
it getting the width of the button  40px

**Comment by AyoubSayah** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/57968451/159916250-8e195c31-7940-4e34-a5c3-5f300926e308.png)
after reading the source code i found that u can pass props to rootprops and they will be passed to popover
